src/index.js

```diff
@@ -63,7 +63,7 @@ const schedule = args.includes('--schedule');
     app.attach(new Notifier(app.serviceDeclarations));
   }
 
-  if (process.env.GITHUB_TOKEN_CREATE_ISSUE) {
+  if (GitHub.isValidToken()) {
```

Comment:
I think here we should not test for token validity. 
The intention here is to say: `if you give me a token, I assume that you want GitHub module to be attached` and not `only if the token is valid, I assume that you want GitHub module to be attached`.

Reply:
You're right, I just found odd to have this environment variable repeated outside of the module